### PR TITLE
Added jsQues

### PR DESCRIPTION
**Pull request by gyanSDE**
Java Script new question added.

### CODE REVIEW

File: javascript/answers/q6.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+                <h4>
+                   What is SetTimeout()?
+                </h4>
+            </div>
+            <br>
+            <br>
+            <div>
+                <strong>Answer:</strong> 
+                <p>
+                    When you setTimeout it becomes asynchronous and it has to wait on the stack to get everything got finished.
```
Comment (arpansac) [inline javascript/answers/q6.html]: Please elaborate on the answer

**Review by arpansac — CHANGES_REQUESTED** [review]

File: javascript/answers/q6.html (hunk 1 above)
Comment (gyanSDE) [inline javascript/answers/q6.html]: 1: Answer elaborated 
2: git add .
3: git commit -m 'elaborate ans'
4: git push origin newq-jsQues
//Anything more  i need to do Sir?

**Comment by gyanSDE** [comment]
Done sir

File: javascript/answers/q6.html (hunk 1 above)
Comment (shivamyth) [reply javascript/answers/q6.html]: Change the name of  answer  HTML file as the format, check the merged questions for example

**Review by shivamyth — CHANGES_REQUESTED** [review]

File: javascript/answers/q6.html (hunk 1 above)
Comment (gyanSDE) [reply javascript/answers/q6.html]: Changed html file name

File: javascript/index.html [hunk 2]
```diff
@@ -113,7 +113,7 @@ <h4>What are escape Sequences?</h4>
                 </a>
             </li>
             <li>
-                <a href="./answers/q6.html">
+                <a href="./answers/Gyan-Kr-q6.html">
```
Comment (gyanSDE) [inline javascript/index.html:120]: Done

**Review by arpansac — DISMISSED** [review]
Looks good. Can you please update it with a little better answer!

**Review by arpansac — APPROVED** [review]

**Pull request merged**